passed
2016-05-13 12:42:47.889
/usr/bin/time --verbose ruby -v -W0 /home/greg/Desktop/src/Open-Table-Explorer/test/unit/shell_command_test.rb
ruby 2.2.4p230 (2015-12-16) [x86_64-linux-gnu]
Loaded suite /home/greg/Desktop/src/Open-Table-Explorer/test/unit/shell_command_test
Started
.......

Finished in 0.044866001 seconds.
------
7 tests, 37 assertions, 0 failures, 0 errors, 0 pendings, 0 omissions, 0 notifications
100% passed
------
156.02 tests/s, 824.68 assertions/s
Run options: --seed 0

# Running:

...$ $?=1
@process_status=#<Process::Status: pid 14895 exit 127>
....Frescue exception in Shell#execute#<Errno::ENOENT: No such file or directory - cd>
/home/greg/Desktop/src/Open-Table-Explorer/app/models/shell_command.rb:114:in `execute'
/home/greg/Desktop/src/Open-Table-Explorer/app/models/shell_command.rb:146:in `initialize'
/home/greg/Desktop/src/Open-Table-Explorer/test/unit/shell_command_test.rb:101:in `new'
/home/greg/Desktop/src/Open-Table-Explorer/test/unit/shell_command_test.rb:101:in `test_03'
/usr/lib/ruby/vendor_ruby/minitest/test.rb:108:in `block (3 levels) in run'
/usr/lib/ruby/vendor_ruby/minitest/test.rb:205:in `capture_exceptions'
/usr/lib/ruby/vendor_ruby/minitest/test.rb:105:in `block (2 levels) in run'
/usr/lib/ruby/vendor_ruby/minitest/test.rb:256:in `time_it'
/usr/lib/ruby/vendor_ruby/minitest/test.rb:104:in `block in run'
/usr/lib/ruby/vendor_ruby/minitest.rb:331:in `on_signal'
/usr/lib/ruby/vendor_ruby/minitest/test.rb:276:in `with_info_handler'
/usr/lib/ruby/vendor_ruby/minitest/test.rb:103:in `run'
/usr/lib/ruby/vendor_ruby/minitest.rb:778:in `run_one_method'
/usr/lib/ruby/vendor_ruby/minitest.rb:305:in `run_one_method'
/usr/lib/ruby/vendor_ruby/minitest.rb:293:in `block (2 levels) in run'
/usr/lib/ruby/vendor_ruby/minitest.rb:292:in `each'
/usr/lib/ruby/vendor_ruby/minitest.rb:292:in `block in run'
/usr/lib/ruby/vendor_ruby/minitest.rb:331:in `on_signal'
/usr/lib/ruby/vendor_ruby/minitest.rb:318:in `with_info_handler'
/usr/lib/ruby/vendor_ruby/minitest.rb:291:in `run'
/usr/lib/ruby/vendor_ruby/minitest.rb:152:in `block in __run'
/usr/lib/ruby/vendor_ruby/minitest.rb:152:in `map'
/usr/lib/ruby/vendor_ruby/minitest.rb:152:in `__run'
/usr/lib/ruby/vendor_ruby/minitest.rb:129:in `run'
/usr/lib/ruby/vendor_ruby/minitest.rb:56:in `block in autorun'
....$ $?=1
@process_status=#<Process::Status: pid 14895 exit 127>
$ ls happyHappyFailFail.junk
@process_status=#<Process::Status: pid 14897 exit 2>
F.F..$ ls happyHappyFailFail.junk
@process_status=#<Process::Status: pid 14897 exit 2>
...$ $?=1
@process_status=#<Process::Status: pid 14895 exit 127>
$ ls happyHappyFailFail.junk
@process_status=#<Process::Status: pid 14897 exit 2>
..$ echo "1 2;3 4"
1 2;3 4
..rescue exception in Shell#execute#<Errno::ENOENT: No such file or directory - cd>
/home/greg/Desktop/src/Open-Table-Explorer/app/models/shell_command.rb:114:in `execute'
/home/greg/Desktop/src/Open-Table-Explorer/app/models/shell_command.rb:146:in `initialize'
/home/greg/Desktop/src/Open-Table-Explorer/test/unit/shell_command_test.rb:96:in `new'
/home/greg/Desktop/src/Open-Table-Explorer/test/unit/shell_command_test.rb:96:in `test_02'
/usr/lib/ruby/vendor_ruby/minitest/test.rb:108:in `block (3 levels) in run'
/usr/lib/ruby/vendor_ruby/minitest/test.rb:205:in `capture_exceptions'
/usr/lib/ruby/vendor_ruby/minitest/test.rb:105:in `block (2 levels) in run'
/usr/lib/ruby/vendor_ruby/minitest/test.rb:256:in `time_it'
/usr/lib/ruby/vendor_ruby/minitest/test.rb:104:in `block in run'
/usr/lib/ruby/vendor_ruby/minitest.rb:331:in `on_signal'
/usr/lib/ruby/vendor_ruby/minitest/test.rb:276:in `with_info_handler'
/usr/lib/ruby/vendor_ruby/minitest/test.rb:103:in `run'
/usr/lib/ruby/vendor_ruby/minitest.rb:778:in `run_one_method'
/usr/lib/ruby/vendor_ruby/minitest.rb:305:in `run_one_method'
/usr/lib/ruby/vendor_ruby/minitest.rb:293:in `block (2 levels) in run'
/usr/lib/ruby/vendor_ruby/minitest.rb:292:in `each'
/usr/lib/ruby/vendor_ruby/minitest.rb:292:in `block in run'
/usr/lib/ruby/vendor_ruby/minitest.rb:331:in `on_signal'
/usr/lib/ruby/vendor_ruby/minitest.rb:318:in `with_info_handler'
/usr/lib/ruby/vendor_ruby/minitest.rb:291:in `run'
/usr/lib/ruby/vendor_ruby/minitest.rb:152:in `block in __run'
/usr/lib/ruby/vendor_ruby/minitest.rb:152:in `map'
/usr/lib/ruby/vendor_ruby/minitest.rb:152:in `__run'
/usr/lib/ruby/vendor_ruby/minitest.rb:129:in `run'
/usr/lib/ruby/vendor_ruby/minitest.rb:56:in `block in autorun'
..rescue exception in Shell#execute#<Errno::ENOENT: No such file or directory - cd>
/home/greg/Desktop/src/Open-Table-Explorer/app/models/shell_command.rb:114:in `execute'
/home/greg/Desktop/src/Open-Table-Explorer/app/models/shell_command.rb:146:in `initialize'
/home/greg/Desktop/src/Open-Table-Explorer/test/unit/shell_command_test.rb:106:in `new'
/home/greg/Desktop/src/Open-Table-Explorer/test/unit/shell_command_test.rb:106:in `test_04'
/usr/lib/ruby/vendor_ruby/minitest/test.rb:108:in `block (3 levels) in run'
/usr/lib/ruby/vendor_ruby/minitest/test.rb:205:in `capture_exceptions'
/usr/lib/ruby/vendor_ruby/minitest/test.rb:105:in `block (2 levels) in run'
/usr/lib/ruby/vendor_ruby/minitest/test.rb:256:in `time_it'
/usr/lib/ruby/vendor_ruby/minitest/test.rb:104:in `block in run'
/usr/lib/ruby/vendor_ruby/minitest.rb:331:in `on_signal'
/usr/lib/ruby/vendor_ruby/minitest/test.rb:276:in `with_info_handler'
/usr/lib/ruby/vendor_ruby/minitest/test.rb:103:in `run'
/usr/lib/ruby/vendor_ruby/minitest.rb:778:in `run_one_method'
/usr/lib/ruby/vendor_ruby/minitest.rb:305:in `run_one_method'
/usr/lib/ruby/vendor_ruby/minitest.rb:293:in `block (2 levels) in run'
/usr/lib/ruby/vendor_ruby/minitest.rb:292:in `each'
/usr/lib/ruby/vendor_ruby/minitest.rb:292:in `block in run'
/usr/lib/ruby/vendor_ruby/minitest.rb:331:in `on_signal'
/usr/lib/ruby/vendor_ruby/minitest.rb:318:in `with_info_handler'
/usr/lib/ruby/vendor_ruby/minitest.rb:291:in `run'
/usr/lib/ruby/vendor_ruby/minitest.rb:152:in `block in __run'
/usr/lib/ruby/vendor_ruby/minitest.rb:152:in `map'
/usr/lib/ruby/vendor_ruby/minitest.rb:152:in `__run'
/usr/lib/ruby/vendor_ruby/minitest.rb:129:in `run'
/usr/lib/ruby/vendor_ruby/minitest.rb:56:in `block in autorun'
.rescue exception in Shell#execute#<Errno::ENOENT: No such file or directory - cd>
/home/greg/Desktop/src/Open-Table-Explorer/app/models/shell_command.rb:114:in `execute'
/home/greg/Desktop/src/Open-Table-Explorer/app/models/shell_command.rb:146:in `initialize'
/home/greg/Desktop/src/Open-Table-Explorer/test/unit/shell_command_test.rb:111:in `new'
/home/greg/Desktop/src/Open-Table-Explorer/test/unit/shell_command_test.rb:111:in `test_05'
/usr/lib/ruby/vendor_ruby/minitest/test.rb:108:in `block (3 levels) in run'
/usr/lib/ruby/vendor_ruby/minitest/test.rb:205:in `capture_exceptions'
/usr/lib/ruby/vendor_ruby/minitest/test.rb:105:in `block (2 levels) in run'
/usr/lib/ruby/vendor_ruby/minitest/test.rb:256:in `time_it'
/usr/lib/ruby/vendor_ruby/minitest/test.rb:104:in `block in run'
/usr/lib/ruby/vendor_ruby/minitest.rb:331:in `on_signal'
/usr/lib/ruby/vendor_ruby/minitest/test.rb:276:in `with_info_handler'
/usr/lib/ruby/vendor_ruby/minitest/test.rb:103:in `run'
/usr/lib/ruby/vendor_ruby/minitest.rb:778:in `run_one_method'
/usr/lib/ruby/vendor_ruby/minitest.rb:305:in `run_one_method'
/usr/lib/ruby/vendor_ruby/minitest.rb:293:in `block (2 levels) in run'
/usr/lib/ruby/vendor_ruby/minitest.rb:292:in `each'
/usr/lib/ruby/vendor_ruby/minitest.rb:292:in `block in run'
/usr/lib/ruby/vendor_ruby/minitest.rb:331:in `on_signal'
/usr/lib/ruby/vendor_ruby/minitest.rb:318:in `with_info_handler'
/usr/lib/ruby/vendor_ruby/minitest.rb:291:in `run'
/usr/lib/ruby/vendor_ruby/minitest.rb:152:in `block in __run'
/usr/lib/ruby/vendor_ruby/minitest.rb:152:in `map'
/usr/lib/ruby/vendor_ruby/minitest.rb:152:in `__run'
/usr/lib/ruby/vendor_ruby/minitest.rb:129:in `run'
/usr/lib/ruby/vendor_ruby/minitest.rb:56:in `block in autorun'
..........

Finished in 0.058435s, 633.1777 runs/s, 1009.6617 assertions/s.

  1) Failure:
ShellCommandsTest#test_success? [/home/greg/Desktop/src/Open-Table-Explorer/test/unit/shell_command_test.rb:144]:
Expected: 127
  Actual: false


  2) Failure:
ShellCommandsTest#test_force_success [/home/greg/Desktop/src/Open-Table-Explorer/test/unit/shell_command_test.rb:161]:
Expected: 0
  Actual: false


  3) Failure:
ShellCommandsTest#test_clear_error_message [/home/greg/Desktop/src/Open-Table-Explorer/test/unit/shell_command_test.rb:149]:
Expected: 0
  Actual: true

37 runs, 59 assertions, 3 failures, 0 errors, 0 skips

Command exited with non-zero status 1
	Command being timed: "ruby -v -W0 /home/greg/Desktop/src/Open-Table-Explorer/test/unit/shell_command_test.rb"
	User time (seconds): 1.03
	System time (seconds): 0.10
	Percent of CPU this job got: 94%
	Elapsed (wall clock) time (h:mm:ss or m:ss): 0:01.20
	Average shared text size (kbytes): 0
	Average unshared data size (kbytes): 0
	Average stack size (kbytes): 0
	Average total size (kbytes): 0
	Maximum resident set size (kbytes): 28828
	Average resident set size (kbytes): 0
	Major (requiring I/O) page faults: 0
	Minor (reclaiming a frame) page faults: 17108
	Voluntary context switches: 402
	Involuntary context switches: 178
	Swaps: 0
	File system inputs: 8
	File system outputs: 8
	Socket messages sent: 0
	Socket messages received: 0
	Signals delivered: 0
	Page size (bytes): 4096
	Exit status: 1
